source/security/private-endpoints.txt

```diff
@@ -98,113 +76,109 @@ connection using either the App Services UI or the Admin API.
+               command you ran in the previous step or on the endpoint details
+               screen of the AWS VPC dashboard. This value starts with ``vpce-``.
+            #. Copy your new VPC Endpoint's ID and paste it into the ``Your VPC
+               Endpoint ID`` field in the App Services UI, then click
+               :guilabel:`Next`.
             #. If this is the first Private Endpoint you are creating, you will 
                see a toggle you can use to either restrict or open all access from 
                requests outside your PrivateLink connection. By default, App 
                Services blocks all requests coming from outside your PrivateLink 
                connection. This setting applies to this endpoint and **all 
                additional endpoints you create**. Refer to :ref:`change access 
                restrictions <change_access_restrictions>` for more information.
+            #. Click :guilabel:`Finish Endpoint Creation`.
 
-            #. Click :guilabel:`Finish Endpoint Creation` to create the endpoint. 
-         
    .. tab::
       :tabid: api
 
       .. procedure::
 
-         .. step:: Create an Interface Endpoint in Your VPC
-
-            To use a Private Endpoint, you must first create a VPC
-            interface endpoint in AWS. You can do this using the
-            :aws:`create-vpc-endpoint
-            </cli/latest/reference/ec2/create-vpc-endpoint>` command in the
-            AWS CLI.
-
-            Modify the following command template with your private
-            endpoint information. Then, copy the command and run it in
-            your terminal:
-
+         .. step:: Create a VPC Interface Endpoint
+
+            a. Call the :admin-api-endpoint:`List VPC Private Endpoint Services
+               <operation/adminListPrivateEndpointServiceInfo>` endpoint.
+            #. Filter the response object to find the entry for your App's
+               deployment region and the endpoint type you want to create.
+
+               .. io-code-block::
+
+                  .. input::
+                     :language: sh
+
+                     curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/security/private_endpoint_service_infos \
+                     -H "Authorization: Bearer <access_token>"
+
+                  .. output::
+                     :language: json
+                     :emphasize-lines: 4-5
+
+                     [
+                     {
+                        "cloud_provider_region": "aws-us-east-1",
+                        "service_name": "com.amazonaws.vpce.us-east-1.vpce-svc-0718bec2c0bdeab5c",
+                        "endpoint_type": "general"
+                     },
+                     ...
+                     ]
+            #. Get the ``service_name`` field from the desired entry in the
+               previous step.
+            #. Create a VPC Endpoint using the AWS CLI `create-vpc-endpoint 
```

Comment:
Hmm, we're saying here to use the AWS CLI to create the endpoint, but this command looks exactly like the one in Step 1 of the UI procedure. Is that one _also_ supposed to be using the AWS CLI? If so, we should probably specify that there. (Around ln 98/99.)

Reply:
Also, if we do expect users to use AWS CLI for _both_ versions of this procedure, should that be listed in the "Before You Begin" section - i.e. "Install and authenticate the AWS CLI"?

Reply:
Yep. Good catch. It should absolutely be in the Before You Begin section.